app/actions/websocket/index.ts

```diff
@@ -95,7 +92,10 @@ async function doReconnect(serverUrl: string, groupLabel?: BaseRequestGroupLabel
         loadConfigAndCalls(serverUrl, currentUserId, groupLabel);
     }
 
-    await deferredAppEntryActions(serverUrl, lastFullSync, currentUserId, currentUserLocale, prefData.preferences, config, license, teamData, chData, initialTeamId, undefined, groupLabel);
+    await deferredAppEntryActions(serverUrl, lastFullSync, currentUserId, currentUserLocale, prefData.preferences, config, license, teamData, chData, meData, initialTeamId, undefined, groupLabel);
+
+    await setLastFullSync(operator, now);
+    setTeamLoading(serverUrl, false);
```

Comment:
Since we're not fetching everything anymore until after deferredAppEntryActions, we should wait until deferredAppEntryActions to complete before setting lastFullSync.